tests/www/apply/test_process.py

```diff
@@ -1951,3 +1951,28 @@ def test_precriber_details_with_older_valid_approval(client, faker):
     )
     # Must display approval status template (tested in many other places)
     assertTemplateUsed(response, template_name="approvals/includes/status.html")
+
+
+@pytest.mark.parametrize("with_inverted_vae_contract", [True, False])
+def test_details_for_geiq_with_inverted_vae_contract(client, with_inverted_vae_contract):
+    # GEIQ: check that contract type is displayed in details
+    job_application = JobApplicationFactory(
+        state=JobApplicationWorkflow.STATE_ACCEPTED,
+        to_siae__kind=SiaeKind.GEIQ,
+        contract_type=ContractType.PROFESSIONAL_TRAINING,
+        inverted_vae_contract=with_inverted_vae_contract,
+    )
+
+    user = job_application.to_siae.members.first()
+    client.force_login(user)
+
+    response = client.get(reverse("apply:details_for_siae", kwargs={"job_application_id": job_application.pk}))
+
+    inverted_vae_text = "associé à une VAE inversée"
+
+    if with_inverted_vae_contract:
+        assertContains(response, job_application.get_contract_type_display())
+        assertContains(response, inverted_vae_text)
+    else:
+        assertContains(response, job_application.get_contract_type_display())
+        assertNotContains(response, inverted_vae_text)
```

Comment:
Proposition _anti-if_ :D
```suggestion
@pytest.mark.parametrize("inverted_vae_contract,expected_predicate", [
    (True, assertContains),
    (False, assertNotContains)
])
def test_details_for_geiq_with_inverted_vae_contract(client, inverted_vae_contract, expected_predicate):
    # GEIQ: check that contract type is displayed in details
    job_application = JobApplicationFactory(
        state=JobApplicationWorkflow.STATE_ACCEPTED,
        to_siae__kind=SiaeKind.GEIQ,
        contract_type=ContractType.PROFESSIONAL_TRAINING,
        inverted_vae_contract=inverted_vae_contract,
    )

    user = job_application.to_siae.members.first()
    client.force_login(user)

    response = client.get(reverse("apply:details_for_siae", kwargs={"job_application_id": job_application.pk}))

    assertContains(response, job_application.get_contract_type_display())
    expected_predicate(response, "associé à une VAE inversée")
```

Reply:
Ah oui, je suis pas un grand habitué des `parametrize`. Merci !